yarpl/include/yarpl/flowable/Flowable.h

```diff
@@ -100,6 +113,16 @@ class Flowable : public virtual Refcounted, public yarpl::enable_get_ref {
 
   Reference<Flowable<T>> observeOn(folly::Executor&);
 
+  template <typename Q>
+  using enableWrapRef =
```

Comment:
Do you use this more than once? Do we need to define it at all?

Reply:
Unrelated, but I prefer `static_assert` always to `enable_if`.

Reply:
It's to help with readability. IMO a complex inline `enable_if` is pretty illegible as the return type for `merge`.

Reply:
@alexmalyshev the `enable_if` is to SFINAE out `merge` when T is not a Flowable

Reply:
I know, my opinion is that SFINAE produces worse errors than a static_assert failure.

*goes off to the hills to shout about C++ Concepts*